src/api/java/baritone/api/Settings.java

```diff
@@ -1216,6 +1218,8 @@ public final Type getType() {
     // here be dragons
 
     Settings() {
+        Registry.BLOCK.getValue(new ResourceLocation("immersive_portals", "portal_placeholder")).ifPresent(blocksToAvoid.defaultValue::add);
```

Comment:
or should this be hardcoded in MovementHelper?

Reply:
LMFAO this is an AMAZINGLY scuffed approach

Reply:
actually beautiful